### ISSUE TITLE
Running the aemanalyser gives an error for an out of the box osgi configuration

### ISSUE THREAD

**Issue by royteeuwen** [adobe/aemanalyser-maven-plugin#101]
### Expected Behaviour
Validation should give no warnings for SDK configurations

### Actual Behaviour

[ERROR] [configuration-api] Configuration org.apache.sling.commons.log.LogManager.factory.config~asyncjssightly: Property org.apache.sling.commons.log.level - Value ERROR does not match provided options

#### Sample Code that illustrates the problem
```
 <plugin>
      <groupId>com.adobe.aem</groupId>
      <artifactId>aemanalyser-maven-plugin</artifactId>
      <version>1.1.8</version>
      <configuration>
          <strictValidation>true</strictValidation>
      </configuration>
      <executions>
          <execution>
              <id>aem-analyser</id>
              <goals>
                  <goal>project-analyse</goal>
              </goals>
          </execution>
      </executions>
  </plugin>
```

**Comment by cziegeler** [comment]
@royteeuwen The problem should be solved with the latest AEM SDK api release, can you please verify?

**Comment by cziegeler** [comment]
Fixed with latest SDK api release
